engines/access/access.h

```diff
@@ -248,9 +269,7 @@ class AccessEngine : public Engine {
 	int _numLines;
 	byte _byte26CB5;
 	int _bcnt;
-	byte *_tempList;
-	int _pictureTaken;
-	//
+	//byte *_tempList;
```

Comment:
If this isn't used anymore, it should be removed

Reply:
Done.